cql3/expr/expression.cc

```diff
@@ -574,9 +575,9 @@ value_list get_IN_values(
     if (auto dv = dynamic_pointer_cast<lists::delayed_value>(t)) {
```

Comment:
GitHub doesn't let me comment on unchanged code, but two lines above this one there's a `using std::views::transform`. Isn't it unused now?

Reply:
I'll remove it.